skopt/utils.py

```diff
@@ -292,3 +295,44 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+def transform_gpdims(dimensions):
```

Comment:
What do you think of making this as `transform_dims` with an `estimator` argument?, then we can relegate the special casing of `estimator` to `transform_dims`